### PR TITLE
Update admin_guide.md

### PR DESCRIPTION
**Pull request by TheVillageGuy**
Added required packages for running (not installed by default on debian 12)

sudo add-apt-repository ppa:ondrej/php

### CODE REVIEW

File: docs/admin_guide.md [hunk 1]
```diff
@@ -20,9 +20,9 @@ This guide is aimed for Debian / Ubuntu distribution servers, but it could run o
 
 ```bash
 sudo apt-get update && sudo apt-get upgrade -y
+sudo apt-get install software-properties-common python3-launchpadlib -y
 sudo add-apt-repository ppa:ondrej/php -y
 sudo apt-get install git redis-server postgresql postgresql-contrib nginx php8.2-common php8.2-fpm php8.2-cli php8.2-amqp php8.2-pgsql php8.2-gd php8.2-curl php8.2-simplexml php8.2-dom php8.2-xml php8.2-redis php8.2-mbstring php8.2-intl unzip -y
-sudo curl -sS https://getcomposer.org/installer -o /tmp/composer-setup.php
```
Comment (melroy89) [inline docs/admin_guide.md:25]: Why remove the composer installation?

**Review by melroy89 — CHANGES_REQUESTED** [review]
You remove composer download step!?

File: docs/admin_guide.md (hunk 1 above)
Comment (TheVillageGuy) [reply docs/admin_guide.md:25]: I must have screwed up let me try again

**Comment by TheVillageGuy** [comment]
> You removed composer download step!?

Had to try ruining it, didn't I!

**Comment by melroy89** [comment]
you could just introduce this step again, and keep the PR open? You didn't need to close the PR!?

**Review by melroy89 — APPROVED** [review]
Lgtm now, thanks!

File: docs/admin_guide.md [hunk 2]
```diff
@@ -132,7 +133,6 @@ Other important `.env` configs:
 KBIN_STORAGE_URL=https://domain.tld/media
 
 # Ubuntu 22.04 installs PostgreSQL v14 by default, Debian 12 PostgreSQL v15 is the default
-POSTGRES_VERSION=14
```
Comment (melroy89) [inline docs/admin_guide.md:135]: You removed `POSTGRES_VERSION=14`??? Maybe you need some sleep 😪  ?

**Review by melroy89 — CHANGES_REQUESTED** [review]
Restore: `POSTGRES_VERSION=14`

File: docs/admin_guide.md (hunk 2 above)
Comment (TheVillageGuy) [reply docs/admin_guide.md:135]: Nah it's a bit early for that yet

**Review by melroy89 — APPROVED** [review]
